README.md

```diff
@@ -1 +1,51 @@
-# dat
+# Delta Acceptance Testing (DAT)
+
+The DAT project makes it easy to generate Delta Lake reference tables and run integration tests to check different implementations of the Delta Lake PROTOCOL behave as expected.
+
+The Delta Lake implementations can clone this repo, generate the reference tables, and implement the suggested integration tests in the language of their choice.
+
+## Installation
+
+You can install the dependencies for this project by running `poetry install`.  Run `poetry shell` to activate the virtual environment.    If `poetry shell` does not activate the environment on your machine, you may need to run this command:
+
+```
+source "$( poetry env list --full-path | grep Activated | cut -d' ' -f1 )/bin/activate"
+```
+
+Run `python -m dat.main` to make sure everything is working properly.
```

Comment:
what is the expected output of this command?

Reply:
Added the expected output, good catch.